### ISSUE TITLE
No tooltip displayed in today's view in Safari 15.5

### ISSUE THREAD

**Issue by ghost** [plausible/analytics#1886]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Plausible Cloud from plausible.io

### Describe the bug

No tooltip is displayed with the number of visitors in today's view in Safari 15.5 (17613.2.7.1.6).

### Expected behavior

View the number of visitors when the mouse is hovered over.

### Screenshots

![screenshot-safari](https://user-images.githubusercontent.com/104916218/166711990-53ebd855-7944-4b3e-aad9-22eccb5bca60.jpg)
Safari

![screenshot-chrome](https://user-images.githubusercontent.com/104916218/166712527-ddee8539-a268-44db-8f58-98c5f3675749.jpg)
Chrome

### Environment

```markdown
- OS: MacOS Monterey 12.4 beta (21F5071b)
- Browser: Safari
- Browser Version: Versión 15.5 (17613.2.7.1.6)
```

**Comment by Vigasaurus** [comment]
>  could this be related to the recent work on the tooltip?

Potentially - but it does reuse all the same logic for showing/hiding as the old version so it should work the same. I don't have the capability to test on Safari though, so unless it can be reproduced on other browsers I likely can't debug it.

**Comment by mikesmith100** [comment]
Note that this only occurs in single day views.

**Comment by MaybeThisIsRu** [comment]
FYI, I'm seeing this error on Safari's console (but not Firefox!):

<img width="1433" alt="Screenshot 2022-05-05 at 11 37 58 AM" src="https://user-images.githubusercontent.com/3974019/166870680-f53dc4c3-cf58-417f-857a-973b8fcbaba1.png">


<img width="977" alt="Screenshot 2022-05-05 at 11 38 05 AM" src="https://user-images.githubusercontent.com/3974019/166870693-0b26e43a-c077-4f30-b26d-bb42ae8cfb2d.png">

**Comment by Vigasaurus** [comment]
Yep, as it turns out - Safari is further proving itself to just be the modern IE. Fixed in #1887

**Comment by metmarkosaric** [comment]
🤣

thanks @MaybeThisIsRu for sharing the details and @Vigasaurus for the fix!
